### PR TITLE
Alexandre

### PR DESCRIPTION
**Pull request by mistfail**
add my tictactoe in vue.js

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<!DOCTYPE html>
```
Comment (Virgiel) [inline public/index.html]: All html files must have the DOCTYPE header, revert this change

File: src/components/alexandre/AlexandreGame.vue [hunk 2]
```diff
@@ -1,15 +1,129 @@
...
+
     export default {
-        name: "AlexandreGame"
+        name: "AlexandreGame",
+        data() {
+            return {
+                isPlayable: true,
+                isPlayerOne: true,
+                table: Array(9).fill(EMPTY_CHAR),
+                victory: false,
```
Comment (Virgiel) [inline src/components/alexandre/AlexandreGame.vue]: use name starting with is- or has- for differentiating boolean variables from others

File: src/components/alexandre/AlexandreGame.vue [hunk 3]
```diff
@@ -1,15 +1,129 @@
...
+                end: false,
+                turn: 1
+            }
+        },
+        computed: {
+            formatPlayerTurn() {
+                if(!this.victory){
+                    return 'Player ' + (this.isPlayerOne ? PLAYER1_CHAR : PLAYER2_CHAR) + ' turn';
+                }
+                return 'Player ' + (!this.isPlayerOne ? PLAYER1_CHAR : PLAYER2_CHAR) + ' win';
```
Comment (Virgiel) [inline src/components/alexandre/AlexandreGame.vue]: Try to use a single line with two elvis operator conditions

File: src/components/alexandre/AlexandreGame.vue [hunk 4]
```diff
@@ -1,15 +1,129 @@
...
+                if (!this.isPlayable || this.table[index] !== EMPTY_CHAR) return;
+
+                const char = this.isPlayerOne ? PLAYER1_CHAR : PLAYER2_CHAR;
+
+                this.table[index] = char;
+                this.isPlayerOne = !this.isPlayerOne;
+                if (isWin(this.table, char)) {
+                    this.victory = true;
+                    this.isPlayable = false;
+                }else{
```
Comment (Virgiel) [inline src/components/alexandre/AlexandreGame.vue]: Typos

File: src/components/alexandre/AlexandreGame.vue [hunk 5]
```diff
@@ -1,15 +1,129 @@
...
+                    this.turn++;
+                }
+                if(this.turn === 10){
```
Comment (Virgiel) [inline src/components/alexandre/AlexandreGame.vue]: Typos

Comment (mistfail) [reply src/components/alexandre/AlexandreGame.vue]: done

File: src/components/alexandre/AlexandreGame.vue (hunk 4 above)
Comment (mistfail) [reply src/components/alexandre/AlexandreGame.vue]: done

File: src/components/alexandre/AlexandreGame.vue (hunk 2 above)
Comment (mistfail) [reply src/components/alexandre/AlexandreGame.vue]: done

File: public/index.html (hunk 1 above)
Comment (mistfail) [reply public/index.html]: done

File: src/components/alexandre/AlexandreGame.vue (hunk 3 above)
Comment (mistfail) [reply src/components/alexandre/AlexandreGame.vue]: done
